ramls/uc.raml

```diff
@@ -197,3 +197,16 @@ types:
             example:
               strict: false
               value: !include examples/uc/uc_settings_patch_422_response.json
+/eholdings/uc-credentials/clientId:
+  displayName: Get Usage Consolidation client id
+  get:
+    description: Get Usage Consolidation client id
+    responses:
+      200:
+        description: OK
+        body:
+          application/vnd.api+json:
```

Comment:
Should it be "text/plain" content-type? As said in the ticket.
Or "application/vnd.api+json" is required type for kb-ebsco?